### ISSUE TITLE
Feature Request: admin_nodeInfo

### ISSUE THREAD

**Issue by MysticRyuujin** [NethermindEth/nethermind#1270]
This is just a request to get admin_nodeInfo() implemented in the JSON RPC. I use it a lot for scripting purposes, information gathering, etc.

**Comment by tkstanczak** [comment]
we can add a version on each module in the module attribute and use it also in the documentation generator

**Comment by tkstanczak** [comment]
also worth reviewing it together with the default enabled modules

**Comment by tkstanczak** [comment]
Done, coming in 1.7.13

values below are numbers (not string) - just the CLI display is wrong

![image](https://user-images.githubusercontent.com/498913/77928547-76b30900-72a0-11ea-99b8-ddf83ddbb084.png)

**Comment by MysticRyuujin** [comment]
So I don't know if this matters or not BUT...

When I query a Geth node the `id` and the `enode` are different...

When I query my Nethermind node they are the same... I feel like the id Geth is returning is something else entirely (it's a different length too), but what, I couldn't say.

E.G.

Geth
```
id         : ae3623ef35c06ab49e9ae4b9f5a2b0f1983c28f85de1ccc98e2174333fdbdf1f
enode      : enode://e1b7e0dc09aae610c9dec8a0bee62bab9946cc27ebdd2f9e3571ed6d444628f99e91e43f4a14d42d498217608bb3e1d1bc8ec2aa27d7f7e423413b851bae02bc@47.186.115.63:30301
```

Nethermind
```
id         : d480513d6dfe03d9b444534973402a5d792b4a333f7ab53613c8231741375af6a972835f73349476c100a12c88aedb476db3d590e0
enode      : enode://d480513d6dfe03d9b444534973402a5d792b4a333f7ab53613c8231741375af6a972835f73349476c100a12c88aedb476db3d590e0f00135181b3287ef1cf5e6@47.186.115.63:30305
```

**Comment by tkstanczak** [comment]
ah, I think it may be the hash of the public key - I will try to change it
to match.

thanks for reporting it!

**Comment by tkstanczak** [comment]
@MysticRyuujin this is fixed now and will be available in 1.8.3+
